internal/core/command/controller/messaging/utils.go

```diff
@@ -25,31 +25,28 @@ import (
 )
 
 // validateRequestTopic validates the request topic by checking the existence of device and device service,
-// returns the internal device request topic and service name to which the command request will be sent.
-func validateRequestTopic(prefix string, deviceName string, commandName string, method string, dic *di.Container) (string, string, error) {
+// returns the service name to which the command request will be sent.
+func validateRequestTopic(deviceName string, dic *di.Container) (string, error) {
```

Comment:
the function name and its purpose is quite different here.
please rename this function to `retrieveServiceNameByDevice`
The go docs should be 
```
// retrieveServiceNameByDevice validates the existence of device and device service,
// returns the service name to which the command request will be sent.
```

Reply:
Updated func name and comment.